searx/infopage/en/about.md

```diff
@@ -32,10 +32,10 @@ own server!
 SearXNG supports [OpenSearch].  For more information on changing your default
 search engine, see your browser's documentation:
 
-- [Firefox]
+- [Firefox] - Behind the link, you will also find some useful instructions for Firefox.
 - [Microsoft Edge] - Behind the link, you will also find some useful instructions
   for Chrome and Safari.
```

Comment:
I think there is a misunderstanding: The Firefox link is only for firefox, while Edge link also has *"useful instructions for Chrome and Safari."*